test/lib/utils.ts

```diff
@@ -0,0 +1,90 @@
+import { evmChains, type EvmChains } from "@infinex/infinex-sdk/sdk/types";
+import { ethers } from "ethers";
+import * as toml from "smol-toml";
+import fs from "node:fs/promises";
+import dotenv from "dotenv";
+
+export { evmChains, type EvmChains } from "@infinex/infinex-sdk/sdk/types";
+export { ethers } from "ethers";
```

Comment:
```
export { evmChains, type EvmChains } from "@infinex/infinex-sdk/sdk/types";
export { ethers } from "ethers";
```

Duplicate

Reply:
Do mean with lines 1 and 2?  These are exports and they're imports.